core/common/src/main/java/alluxio/network/protocol/RPCBlockReadResponse.java

```diff
@@ -24,6 +24,8 @@
 
 /**
  * This represents the response of a {@link RPCBlockReadRequest}.
+ *
+ * Since 1.4.1, this also represents the response of a {@link RPCUnderFileSystemBlockReadRequest}.
```

Comment:
do we plan to release this as 1.4.1? I would use 1.5 here (as 1.4.1 in my mind indicates a patch release of 1.4)